### PR TITLE
Add external NSC example

### PR DESCRIPTION
**Pull request by glazychev-art**
Issue: https://github.com/networkservicemesh/cmd-nsc-simple-docker/issues/1

Signed-off-by: Artem Glazychev <artem.glazychev@xored.com>

### CODE REVIEW

File: examples/k8s_monolith/external_nsc/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# NSC as external docker container
```
Comment (edwarnicke) [inline examples/k8s_monolith/external_nsc/README.md:1]: I am curious why this is in the k8s_monolith example?  We should be able to use  simple docker nsc as a client for any interdomain Network Service use case, not just the breaking up the monolith case.

File: examples/k8s_monolith/README.md [hunk 2]
```diff
@@ -1,45 +1,20 @@
 # Breaking up an application monolith with NSM
 
-**_This example only works with kind kubernetes cluster_**
+**_These examples only work with [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) kubernetes cluster_**
```
Comment (edwarnicke) [inline examples/k8s_monolith/README.md:3]: Why do these examples only work with kind?

**Comment by glazychev-art** [comment]
@edwarnicke 
Do you have any new comments?

**Comment by denis-tingaikin** [comment]
Could you fix linkchecker? Note: you could use the latest revision d2fb9ea52963949ba70d8dc373ab76d99f54b633

**Comment by denis-tingaikin** [comment]
@edwarnicke we're merging this to test on ci. Feel free to add comments here and we'll resolve them in separate PR.

**Pull request merged**